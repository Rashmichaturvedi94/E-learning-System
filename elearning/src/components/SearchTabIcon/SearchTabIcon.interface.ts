import { TabBarIconProps } from 'components/TabBarIcon/TabBarIcon.interface';

export type SearchTabIconProps = Omit<TabBarIconProps, 'Icon'>;
