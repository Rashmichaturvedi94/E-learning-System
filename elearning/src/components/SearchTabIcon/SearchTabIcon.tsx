import React, { FC } from 'react';
import { TabBarIcon } from 'components/TabBarIcon';
import { SearchIcon } from 'components/Icon';
import Svg from 'react-native-svg';
import { SearchTabIcon as SearchTabIconComponent } from './SearchTabIcon.styles';
import { SearchTabIconProps } from './SearchTabIcon.interface';

export const SearchTabIcon: FC<SearchTabIconProps> = ({
  focused,
  ...iconProps
}) => (
  <TabBarIcon
    focused={focused}
    Icon={SearchIcon}
    width={32}
    height={32}
    {...iconProps}
  />
);
