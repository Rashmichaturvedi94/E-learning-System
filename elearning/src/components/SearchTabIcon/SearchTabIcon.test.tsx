import React from 'react';
import { render } from '@testing-library/react-native';
import { SearchTabIcon } from './SearchTabIcon';

describe('components/SearchTabIcon', () => {
  it('should render', () => {
    const component = render(<SearchTabIcon />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
