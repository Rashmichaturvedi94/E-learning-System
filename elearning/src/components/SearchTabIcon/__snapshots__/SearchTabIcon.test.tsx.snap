// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/SearchTabIcon should render 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "opacity": 0.6,
      },
    ]
  }
>
  <SvgMock
    height={32}
    width={32}
  />
</View>
`;
